control-plane/roles/gardener/templates/shoot-cert-service/controller-deployment.yaml

```diff
@@ -15,7 +15,7 @@ providerConfig:
       defaultIssuer:
         restricted: true # restrict default issuer to any sub-domain of shoot.spec.dns.domain
         acme:
-          email: cert-expiry@metal-pod.io
-          server: https://acme-v02.api.letsencrypt.org/directory
+          email: "{{ gardener_cert_management_issuer_email }}"
+          server: "{{ gardener_cert_management_issuer_server }}"
```

Comment:
Can we default this to value to `https://acme-v02.api.letsencrypt.org/directory` to reduce required actions for operators?

Reply:
Also looks like deployment would fail without having these variables defined. So we need to make the email required and should assert that it is defined.